package.json

```diff
@@ -4,10 +4,10 @@
   "description": "Convert numbers to words, in multiple languages",
   "main": "dist/n2words.min.js",
```

Comment:
Do we need to add "type": "module" ?

Reply:
Actually, that won't be necessary as `.mjs` forces that specific file to be loaded with the ESM loader. I'm trying to avoid treating `.js` as module due to /dist being CJS.